integration_tests/pages/challenges/selectChallengeCategoryPage.ts

```diff
@@ -0,0 +1,15 @@
+import Page, { PageElement } from '../page'
+import ChallengeCategory from '../../../server/enums/challengeCategory'
+
+export default class SelectChallengeCategoryPage extends Page {
+  constructor() {
+    super('create-challenge-select-category')
+  }
+
+  selectCategory(value: ChallengeCategory): SelectChallengeCategoryPage {
```

Comment:
As commented earlier, the method arg now needs to be of type `ChallengeType` (and also on L14)